hack/verify-govet.sh

```diff
@@ -22,7 +23,7 @@ function govet_blacklist_contains() {
 }
 
 for test_dir in $(os::util::list_go_src_dirs); do
-	if ! result="$(go tool vet -shadow=false -printfuncs=Info,Infof,Warning,Warningf "${test_dir}" 2>&1)"; then
+	if ! result="$(go vet -printfuncs=Info,Infof,Warning,Warningf "${test_dir}/..." 2>&1)"; then
```

Comment:
./... is apparently the syntax for recursion.  No idea how this was working before, frankly.